Cargo.toml

```diff
@@ -143,6 +143,7 @@ arrow-cast-iceberg = { package = "arrow-cast", version = "52" }
 # After apache/iceberg-rust#411 is merged, we move to the upstream version.
 iceberg = { git = "https://github.com/risingwavelabs/iceberg-rust.git", rev = "0c6e133e6f4655ff9ce4ad57b577dc7f692dd902" }
 iceberg-catalog-rest = { git = "https://github.com/risingwavelabs/iceberg-rust.git", rev = "0c6e133e6f4655ff9ce4ad57b577dc7f692dd902" }
+opendal = "0.47"
```

Comment:
Is there a way to ensure that at least 0.47.2 is used? Version before 0.47.2 will break GCS.

Reply:
Cargo lock shows that we still use 0.47.2. BTW, `opendal = "0.47` means always use the latest version if we run `cargo update`. So I think it is fine.